scanner/src/main/kotlin/scanners/fossid/Utils.kt

```diff
@@ -28,7 +28,7 @@ import org.ossreviewtoolkit.model.config.PathExclude
 
 private val DIRECTORY_REGEX = "(?<directory>(?:\\w+/)*\\w+)/?(?<starstar>\\*\\*)?".toRegex()
 private val EXTENSION_REGEX = "\\*\\.(?<extension>\\w+)".toRegex()
-private val FILE_REGEX = "(?<file>[\\w.]+)".toRegex()
+private val FILE_REGEX = "(?<file>[^/]+)".toRegex()
```

Comment:
Please add to the commit message body: "Generally allow all characters except for a slash for file names now."